config/environments/development.rb

```diff
@@ -47,7 +47,7 @@
   # Raise exceptions for disallowed deprecations.
   config.active_support.disallowed_deprecation = :raise
 
-  # Tell Active Support which deprecation messages to disallow.
```

Comment:
what is this?

You should review carefully while replacing messages with notes. You should only replace if that code is relevant to yours not on all other places

Reply:
fixed